README.md

```diff
@@ -1 +1,67 @@
-Tools for Wizzilab products
+# AppLink via python
+
+Python scripts to communicate with Wizzilab's AppLink. 
+
+- watch_reports.py allows to display reports coming from sensors
+- read_example.py allows to read a property of a device 
+- write_example.py shows how to configure a property of a device. 
+
+## Installation and example
+
+In order to use these scripts you just need to install the paho-mqtt module.
+
+```
+pip3 install paho-mqtt
+```
+
+Once this is done, it is possible to run the scripts, -h option is available to get a list of the required parameters. The watch_reports.py script requires the login credentials that can be found on the Dash7Board the company's page in the AppLink section, for example to connect to the queue with the following login credentials  
```

Comment:
`Once this is done, it is possible to run the scripts, -h option is available to get a list of the required parameters.` Sounds like 2 sentences missing a terminal dot at the end of `run the scripts`.
You should probably split the sentence into 2 sentences there.

In the same way, the next sentence feels like `in the AppLink section,` should be the end of the sentence.